README.md

```diff
@@ -39,6 +39,46 @@ Na dowolnym systemie wspierającym konteneryzację Docker można uruchomić cał
 docker compose up --build
 ```
 
+## Wymagania funkcjonalnie:
+
+- Użytkownik będzie w stanie utworzyć własne konto
+- Użytkownik będzie w stanie zalogować się na swoje konto, zmienić hasło i je usunąć
+- Użytkownik będzie w stanie wysyłać i pobierać pliki
+- Użytkownik będzie w stanie udostępniać pliki innemu użytkownikowi
+- Użytkownik będzie w stanie tworzyć pliki prywatne, które są niewidoczne dla innych użytkowników
+- Użytkownik będzie w stanie kategoryzować pliki oraz nadawać im tagi
+- Użytkownik będzie w stanie wyszukiwać pliki w bazie (do których ma dostęp)
+    - po nazwie
+    - po typie (.pdf, .zip, .txt)
+    - po zawartości (w przypadku plików tekstowych)
+    - po rozmiarze
+    - po dacie modyfikacji
+    - po dacie utworzenia
+- Użytkownik będzie w stanie wykonywać podstawowe operacje na plikach, takie jak:
+    - usunięcie pliku
+    - zmienianie nazwy
+    - zmiana miejsca w dysku
+- Użytkownik będzie w stanie tworzyć foldery w strukturze drzewiastej i umieszczać tam pliki
+- Użytkownik będzie w stanie szyfrować pliki/foldery hasłem
+- Użytkownik będzie w stanie zobaczyć własną ewidencję (ślad cyfrowego aktywności tzn. ile było pobrać, kiedy pobrał)
+- Użytkownik będzie w stanie wyeksportować stan swojego dysku (stan obejmuje strukturę dysku, zaszyfrowane pliki, tagi
+  oraz kategorie)
+- Użytkownik będzie w stanie zaimportować stan dysku z pliku
```

Comment:
I'm not sure if we're gonna be able to add it in such a short period of time

Reply:
what's this then [check bullet point no. 3](https://github.com/maksnowak/FileSharingSystem?tab=readme-ov-ile#opis-projektu)

Reply:
mea culpa